packages/fabric8-tenant-che/src/main/fabric8/template.yml

```diff
@@ -11,4 +11,10 @@ parameters:
 - name: KEYCLOAK_OSO_ENDPOINT
   value: https://sso.prod-preview.openshift.io/auth/realms/fabric8/broker/openshift-v3/token
 - name: KEYCLOAK_GITHUB_ENDPOINT
-  value: https://auth.prod-preview.openshift.io/api/token?for=https://github.com
\ No newline at end of file
+  value: https://auth.prod-preview.openshift.io/api/token?for=https://github.com
+- name: CHE_KEYCLOAK_AUTH__SERVER__URL
+  value: https://sso.prod-preview.openshift.io/auth
```

Comment:
Why is CheHost expecting half a URL here?

Reply:
@davidfestal could you please clarify ?

Reply:
This URL with `auth` at the end is the Keycloak auth endpoint. This comes initially from the settings that we find in the Keycloak documentation. See:
- the `auth-server-url` setting [here](http://www.keycloak.org/docs/latest/securing_apps/index.html#_java_adapter_config) for Java
- and the `url` setting [here](http://www.keycloak.org/docs/latest/securing_apps/index.html#_javascript_adapter) for Javascript